pages/subscriptions-revamp.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from '@apollo/react-hoc';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { generateNotFoundError } from '../lib/errors';
+import { API_V2_CONTEXT, gqlV2 } from '../lib/graphql/helpers';
+
+import AuthenticatedPage from '../components/AuthenticatedPage';
+import Container from '../components/Container';
+import ErrorPage from '../components/ErrorPage';
+import { Flex } from '../components/Grid';
+import Loading from '../components/Loading';
+import StyledTag from '../components/StyledTag';
+import SubscriptionsContainer from '../components/subscriptions-v2/SubscriptionsContainer';
+import { H2 } from '../components/Text';
+import { withUser } from '../components/UserProvider';
+
+const subsRevampPageQuery = gqlV2`
```

Comment:
```suggestion
const subsRevampPageQuery = gqlV2/* GraphQL */ `
```

Small trick so Prettier can detect and format the GraphQL query.